### PR TITLE
Document 'O' keyboard shortcut

### LINKED ISSUE #1236: Keyboard shortcut to hide containerized processes not documented
The `O` keyboard shortcut is not documented in the `F1` help. Could this also be added?
Same for the man page.

_Originally posted by @m0vie in https://github.com/htop-dev/htop/issues/1062#issuecomment-1537134535_

### PR DESCRIPTION
**Pull request by valdaarhun**
Closes #1236.

Document the 'O' keyboard shortcut in the man page and `F1` help screen.

Reported by @m0vie.

### CODE REVIEW

**Comment by valdaarhun** [comment]
> Where are the changes to the offset for the right column coming from?

The `-es` in "processes" is replaced with the characters in the right column when using the original offset. Increasing the offset by two prevents that.

File: Action.c [hunk 1]
```diff
@@ -606,6 +606,7 @@ static const struct {
    { .key = "      u: ",  .roInactive = false, .info = "show processes of a single user" },
    { .key = "      H: ",  .roInactive = false, .info = "hide/show user process threads" },
    { .key = "      K: ",  .roInactive = false, .info = "hide/show kernel threads" },
+   { .key = "      O: ",  .roInactive = false, .info = "hide/show containerized processes" },
```
Comment (fasterit) [inline Action.c:609]: How about "hide/show processes in containers" ?
Same length, seems more understandable / correct English.

**Comment by fasterit** [comment]
> 



> > Where are the changes to the offset for the right column coming from?
> 
> The `-es` in "processes" is replaced with the characters in the right column when using the original offset. Increasing the offset by two prevents that.

We're at 88 columns width now, this change makes it 90.
Somebody did not pay attention that we stay within 80 columns, so ... seems 90 is fine now as well.
We just should make sure it does not continue to grow beyond that...

File: Action.c (hunk 1 above)
Comment (valdaarhun) [reply Action.c:609]: Looks good to me.

**Pull request closed without merging**